docs/app/docs/theming/[slug]/components/shadows/shadow-tokens.tsx

```diff
@@ -0,0 +1,55 @@
+'use client'
+
+import { Box, Grid } from '@cerberus-design/react'
```

Comment:
Great contribution! The React lib doesn't export the `Box` component, that instead should come from the local `styled-system/jsx` location.